adds reminder to user when submitting an issue (#489)
shiftkey/desktop: .github/ISSUE_TEMPLATE/bug_report.md

A clear and concise description of what the bug is.

<!-- Important: Read First -->

> <!-- if you are using the appimage/deb/rpm from the  release page  make sure to use the very latest version  that is available  check releases: https://github.com/shiftkey/desktop/releases   if the issue still occurs in the latest release please  continue  -->

Reviewer: ```suggestion
<!-- If you are using the appimage/deb/rpm from the release page make sure you have the very latest version that is available. Check the releases page to see if there are new updates available: https://github.com/shiftkey/desktop/releases -->
```
